src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrivals/Arrival.kt

```diff
@@ -74,7 +75,13 @@ data class PotentialMatch(
   val sex: String,
 
   @JsonIgnore
-  val isCurrentPrisoner: Boolean
+  val isCurrentPrisoner: Boolean,
+
+  @Schema(description = "Description of arrival type", example = "ACTIVE IN-ADM-MDI")
+  val arrivalTypeDescription: String,
```

Comment:
Added the two new fields to single prisoner and potential matches - it might become useful if we need to provide extra hints to the user to select a match. It also helps simplify (my) mental model to keep prisoner and potential matches in parity!